### PR TITLE
fix: Sanitize unhandled version values

### LINKED ISSUE #752: Sanitize unhandled version values
### Context information

application version: 
device: 
browser: 
operating system: 

### Steps to reproduce
go to any page. Occasionally it will respond with 500.
It's mainly an issue on localhost, but may appear at any stage

### What is expected?
Node and MDW versions are correctly displayed

### What is actually happening?
image

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #752

## Demo
displays as expected

https://github.com/aeternity/aescan/assets/15363559/351d6968-7a3d-44be-93ca-909070b7fde2

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] Sanitize unhandled version values

### CODE REVIEW

**Review by Liubov-crypto — COMMENTED** [review]
I have a question: why I can't see Listed Tokens? (the same on prod)

tok

File: src/components/TheFooter.vue [hunk 1]
```diff
@@ -52,14 +52,14 @@
               class="footer__version">
               NODE VERSION
               <app-link :to="NODE_RELEASES_URL">
-                v{{ nodeStatus.nodeVersion }}
+                v{{ nodeStatus?.nodeVersion }}
```
Comment (michele-franchi) [inline src/components/TheFooter.vue:55]:
```suggestion
                {{ formatNullable(nodeStatus.nodeVersion) }}
```

Might be an opportunity to get rid of the extra "v" and also make it use `formatNullable` for this specific case?

File: src/components/TheFooter.vue [hunk 2]
```diff
@@ -52,14 +52,14 @@
...
               </app-link>
             </div>
             <div
               class="footer__version">
               MIDDLEWARE VERSION
               <app-link :to="MDW_RELEASES_URL">
-                v{{ middlewareStatus.mdwVersion }}
+                v{{ middlewareStatus?.mdwVersion }}
```
Comment (michele-franchi) [inline src/components/TheFooter.vue:62]: Same as suggested above, "v" should be removed also for "ÆSCAN VERSION".

**Review by michele-franchi — CHANGES_REQUESTED** [review]

**Comment by michele-franchi** [comment]
> I have a question: why I can't see Listed Tokens? (the same on prod)
> 
> tok

Asked @mmpetarpeshev to have a look at it, it's a DEX issue.

**Comment by janmichek** [comment]
> I have a question: why I can't see Listed Tokens? (the same on prod)
> 
> tok

Already created issue to trrack it https://github.com/aeternity/aescan/issues/765

File: src/components/TheFooter.vue (hunk 2 above)
Comment (janmichek) [reply src/components/TheFooter.vue]: removed

File: src/components/TheFooter.vue (hunk 1 above)
Comment (janmichek) [reply src/components/TheFooter.vue]: fixed

**Pull request merged**